graphql_api/types/plan/plan.py

```diff
@@ -98,3 +96,39 @@ def resolve_plan_user_count(plan_service: PlanService, info) -> int:
 @sync_to_async
 def resolve_has_seats_left(plan_service: PlanService, info) -> bool:
     return plan_service.has_seats_left
+
+
+@plan_bindable.field("isEnterprisePlan")
+@sync_to_async
+def resolve_is_enterprise_plan(plan_service: PlanService, info) -> bool:
+    return plan_service.is_enterprise_plan
```

Comment:
where are these `PlanService.is_*` attributes defined, I couldn't find them in shared.

Reply:
LOL I totally forgot that we swapped over that stuff to shared, let me open up a new PR for that part. It worked on my local because I modified shared directly